### PR TITLE
Removed DenyEscalatingExec from the list of default admission controllers

### PR DESCRIPTION
**Pull request by ArtfulCoder**
#15035
#15081

### CODE REVIEW

**Comment by bgrant0607** [comment]
LGTM

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -201,8 +201,11 @@ function set_service_accounts {
 
 function start_apiserver {
     # Admission Controllers to invoke prior to persisting objects in cluster
-    ADMISSION_CONTROL=NamespaceLifecycle,NamespaceAutoProvision,LimitRanger,SecurityContextDeny,ServiceAccount,DenyEscalatingExec,ResourceQuota
-
+    if [[ -z "${ALLOW_SECURITY_CONTEXT}" ]]; then
```
Comment (bgrant0607) [inline hack/local-up-cluster.sh]: Was the addition of this if statement in a PR that was not cherrypicked?

Comment (ArtfulCoder) [inline hack/local-up-cluster.sh]: also cherry-picks #15081
let me know if you want me to move it out

**Comment by ArtfulCoder** [comment]
Release notes: By default, users can now use kubectl to exec into a privileged pod. If the cluster admin wants to disable this feature, they need to explicitly start APIServer with "DenyEscalatingExec" in the admission-control flag.

**Pull request merged**